File: packages/aws-cdk/lib/init-templates/v1/sample-app/csharp/src/%name.PascalCased%/%name.PascalCased%Stack.template.cs
using Amazon.CDK;
using Amazon.CDK.AWS.SNS;
using Amazon.CDK.AWS.SNS.Subscriptions;
using Amazon.CDK.AWS.SQS;

namespace %name.PascalCased%
{
    public class %name.PascalCased%Stack : Stack
    {
        internal %name.PascalCased%Stack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
        {
             // The CDK includes built-in constructs for most resource types, such as Queues and Topics.
            var queue = new Queue(this, "%name.PascalCased%Queue", new QueueProps
            {
                VisibilityTimeout = Duration.Seconds(300)
            });

            var topic = new Topic(this, "%name.PascalCased%Topic");

            topic.AddSubscription(new SqsSubscription(queue));
        }
    }
}

File: packages/@aws-cdk/tmp-toolkit-helpers/resources/init-templates/sample-app/fsharp/src/%name.PascalCased%/%name.PascalCased%Stack.template.fs
namespace %name.PascalCased%

open Amazon.CDK
open Amazon.CDK.AWS.SNS
open Amazon.CDK.AWS.SNS.Subscriptions
open Amazon.CDK.AWS.SQS

type %name.PascalCased%Stack(scope, id, props) as this =
    inherit Stack(scope, id, props)

    let queue = Queue(this, "%name.PascalCased%Queue", QueueProps(VisibilityTimeout = Duration.Seconds(300.)))

    let topic = Topic(this, "%name.PascalCased%Topic")
    do topic.AddSubscription(SqsSubscription(queue)) |> ignore
